inc/Engine/Media/Lazyload/Subscriber.php

```diff
@@ -15,7 +15,7 @@
  * @since 3.3
  */
 class Subscriber implements Subscriber_Interface {
-	const SCRIPT_VERSION = '16.1';
+	const SCRIPT_VERSION = '17.4';
```

Comment:
@Tabrisrp Currently latest version is 17.5 as per https://github.com/verlok/vanilla-lazyload/releases, Do we need to use this? (not sure if we can have an option where this can be auto updated always as per latest version whatever)

Reply:
Updated the script version to 17.5. We can't auto-update it, we have to manually handle it.